app/src/main/java/com/rkm/tasky/network/model/response/LoginUserResponse.kt

```diff
@@ -9,3 +10,14 @@ data class LoginUserResponse(
     val userId: String,
     val accessTokenExpirationTimestamp: Long
 )
+
+fun LoginUserResponse.asAuthInfo(): AuthInfo {
+    return AuthInfo(
+        accessToken =  this.accessToken,
+        refreshToken = this.refreshToken,
+        fullName = this.fullName,
+        userId = this.userId,
+        email = "",
+        accessTokenExpirationTimestamp = this.accessTokenExpirationTimestamp
+    )
+}
```

Comment:
Becareful here. You’ve now allowed your app to be in an invalid state by setting the AuthInfo without an empty `email` value. When you have an `AuthInfo` object, you expect all the fields to be there. To avoid this here, pass in the email to the extension function.

```
fun LoginUserResponse.asAuthInfo(email: String): AuthInfo {
  return AuthInfo(
    accessToken =  this.accessToken,
    refreshToken = this.refreshToken,
    fullName = this.fullName,
    userId = this.userId,
    email = email,
    accessTokenExpirationTimestamp = this.accessTokenExpirationTimestamp
  )
}
```